Dockerfile

```diff
@@ -2,18 +2,22 @@ FROM java:openjdk-8-jre
 
 MAINTAINER Eugene Volchek <evolchek@klika-tech.com>
 
-ENV UPSOURCE_VERSION 3.0.4291
-VOLUME ["/opt/Upsource/conf", "/opt/Upsource/data", "/opt/Upsource/logs", "/opt/Upsource/backups"]
+ENV UPSOURCE_VERSION 3.0.4396
+
 WORKDIR /opt
-RUN mkdir -p /home/upsource \
+RUN    mkdir -p /opt/Upsource \
+    && mkdir -p /home/upsource \
 	&& groupadd -g 999 upsource \
 	&& useradd -u 999 -g upsource -d /home/upsource upsource \
 	&& chown -R upsource:upsource /home/upsource \
 	&& wget -nv http://download.jetbrains.com/upsource/upsource-$UPSOURCE_VERSION.zip \
-	&& unzip upsource-$UPSOURCE_VERSION.zip -d /opt/Upsource \
+	&& unzip upsource-$UPSOURCE_VERSION.zip \
+	&& mv upsource-$UPSOURCE_VERSION/* /opt/Upsource \
```

Comment:
I guess we could rename the dir with `mv upsource-$UPSOURCE_VERSION /opt/Upsource` and omit lines 8 and 16